content/docs/contributing.md

```diff
@@ -4,15 +4,17 @@ description: "Contributing to Open Source"
 draft: false
 ---
 
-Participation in open source projects is highly encouraged, and the Open Source Program Office wants to make this process as simple as possible. All Saucers are expected to follow common sense regarding the type of information and code is contributed to upstream open source projects. If you are in doubt, please reach out to the OSPO or ask your manager for direction.
+Participation in open source projects is highly encouraged, and the Open Source Program Office wants to make this process as simple as possible. Saucers are responsible to preserve our valuable information and code. OSPO and your manager can help you understand and determine what is open sourceable and what must remain private. In doubt, always ask.
 
 ### Summary
 
-We highly encourage contributions to existing open source projects. There are some requirements before you start patching, and guides to help you figure out the right path:
+We highly encourage contributions to [existing open source projects](https://opensource.saucelabs.com/projects/).
 
-- Patches generally don't require any review from the Open Source Program Office
+There are some requirements before you start patching, and guides to help you figure out the right path:
+
+- [Patches](https://semver.org/) generally don't require any review from the Open Source Program Office
```

Comment:
The original meaning is not tied to `semver`, but to the general meaning of what a patch [is](https://en.wikipedia.org/wiki/Patch_(computing)).

Reply:
Makes super sense! I will change the link :) thank you for checking!